eagleproject/core/views.py

```diff
@@ -158,12 +158,44 @@ def dashboard(request):
 
     return _cache(20, render(request, "dashboard.html", locals()))
 
-def _get_strat_layout(assets):
+def _get_strat_holdings(assets):
     all_strats = {}
     for (strat_key, strat) in STRATEGIES.items():
         total = 0
         holdings = []
 
+        for asset in assets:
+            if not asset.symbol in strat.get("SUPPORTED_ASSETS", DEFAULT_ASSETS):
+                continue
+            balance = asset.get_strat_holdings(strat_key)
+            holdings.append((asset.symbol, balance))
+            total += balance
+
+        if strat_key == "ousd_metastrat":
+            # Assume that half of holdings in ousd_metastrat is printed OUSD
+            ousd_holding = total / 2
```

Comment:
nit: maybe it be cool to define the "2" as a OUSD_PRINT_FACTOR constant, to have more verbosity on what it does and is easier to change in the future if we ever need to

Reply:
Good point, Noted. Will change it the next time I'm working on this function